src/main/resources/application.properties

```diff
@@ -2,4 +2,4 @@ spring.thymeleaf.cache=false
 spring.thymeleaf.enabled=true 
 spring.thymeleaf.prefix=classpath:/templates/
 spring.thymeleaf.suffix=.html
-server.port=5000
+server.port=8080
```

Comment:
I saw this with your README change, is this going to need to be changed back for deployment to AWS later?

Reply:
Yup! Just need to change it before I package it and deploy
Hopefully with CI/CD setup we can have it be an env var and automatically deploy latest merge